src/components/SelectionList/selectionListPropTypes.js

```diff
@@ -30,33 +52,25 @@ const userListItemPropTypes = {
         rightElement: PropTypes.element,
 
         /** Avatar for the user */
```

Comment:
This comment was for `avatar` property - is it still applying for the new property `icons`?

Reply:
Yep, still applies, but I guess I can make it more clear by matching the key name